src/vis/vis.js

```diff
@@ -427,6 +408,59 @@ var VisModel = Backbone.Model.extend({
 
   isLoading: function () {
     return this.get('loading');
+  },
+
+  /**
+   * "Load" existing analyses from the viz.json. This will generate
+   * the analyses graphs and index analysis nodes in the
+   * collection of analysis
+   */
+  _createAnalysisNodes: function (analysesDefinition) {
+    var analysisNodes = {};
```

Comment:
rename this to `analyses` for being consistent with older code

Reply:
Use analysisNode**s** instead